src/test/java/teammates/test/BaseTestCaseWithPostgresSqlDatabaseAccess.java

```diff
@@ -0,0 +1,51 @@
+package teammates.test;
+
+import teammates.common.datatransfer.sqlattributes.CourseAttributes;
+import teammates.common.datatransfer.sqlattributes.EntityAttributes;
+import teammates.common.util.JsonUtils;
+
+public abstract class BaseTestCaseWithPostgresSqlDatabaseAccess extends BaseTestCase{
```

Comment:
```suggestion
public abstract class BaseTestCaseWithSqlDatabaseAccess extends BaseTestCase {
```
Same here; also, missing space.